### PR TITLE
Add changelog as text file in repository

### PR DESCRIPTION
**Pull request by jordanperr**
Create a changelog file in the root of the repository. I would like feedback on the changelog's format, as well as on the releases and details included in the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Addition of reanalysis schemas to the Sphinx documentation
+- Easy import of EIA data using new module: Metadata_Fetch
+- Updated contributing.md document
+- Quality checks for reanalysis data
+- Improved installation instructions
+- Integration tests are now performed in CI
+- Performed PEP8 linting
+
+## [1.0] - 2018-12-06
+- Complete refactor of many analysis and toolkit modules.
```
Comment (Dynorat) [inline CHANGELOG.md:25]: line 25: for what purpose?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- EYA Gap Analysis
+- Uncertainty quantification
+- Engie example data, and complete update of example notebooks
+- Switch to standard BSD-3 Clause license
+- Automated quality control method
```
Comment (Dynorat) [inline CHANGELOG.md:9]: say a little more on this one.  this is the switch to integrated github tools?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- EYA Gap Analysis
+- Uncertainty quantification
+- Engie example data, and complete update of example notebooks
```
Comment (Dynorat) [inline CHANGELOG.md:7]: Suggested: Implemented open source Engie example data, and complete update of example notebooks

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- EYA Gap Analysis
```
Comment (Dynorat) [inline CHANGELOG.md:5]: Suggested: Energy Yield Analysis (EYA) to Operational Assessment(OA) Gap Analysis method

**Comment by jordanperr** [comment]
@Dynorat please re-review

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- Energy Yield Analysis (EYA) to Operational Assessment(OA) Gap Analysis method
+- Uncertainty quantification
```
Comment (ejsimley) [inline CHANGELOG.md:6]: UQ has always been a part of AEP analysis. Do you mean uncertainty quantification for electrical losses and longterm turbine gross energy (Nicola's recent PR)? This is still in develop though. Soon we can also add "Augmented AEP method with variable temporal resolution, machine learning methods, and improved uncertainty quantification" too, but the PR is still open.

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- Implemented open source Engie example data, and complete update of example notebooks
```
Comment (ejsimley) [inline CHANGELOG.md:7]: Is the unreleased category intended to describe the develop branch or just master? These additions are or will soon be part of develop, but not on master yet. Unless if this is referring to the old API method of accessing Engie data.

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+- Switch to standard BSD-3 Clause license
+- Automated quality control method to assist with data ingestion. Tools in this method include daylight savings time change detection and identification of the diurnal cycle.
+- Electrical losses method
```
Comment (ejsimley) [inline CHANGELOG.md:10]: Another item to add is: Method for estimating long-term turbine gross energy (excluding downtime and underperformance losses)

File: CHANGELOG.md (hunk 6 above)
Comment (jordanperr) [reply CHANGELOG.md:7]: develop branch

**Comment by jordanperr** [comment]
[UNRELEASED] is meant to be for changes that are in the repository but not yet tagged as a release. This will cover everything in the develop branch which is new since the last release. This section is meant to be updated as new PRs of significance are merged.

**Pull request merged**